kolibri/plugins/facility/assets/src/views/users/common/UsersTableToolbar/index.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+
+  <component
+    :is="layoutComponent"
+    :title="title"
+    :hasSelectedUsers="hasSelectedUsers"
+    :showUsersTable="showUsersTable"
+  >
+    <template #headerActions>
+      <slot name="headerActions"></slot>
+    </template>
```

Comment:
This is a great pattern for handling conditional/divergent layouts that include the same basic components!